src/main/java/core/basesyntax/service/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CsvFileReaderImpl implements FileReader {
```

Comment:
Why **Csv**FileReader? I do not see any Csv-specific in the code. This class can read any file.